doc/basics-reference.markdown

```diff
@@ -374,7 +374,7 @@ Property access uses the `.` macro.
 If you wish you could just write those as `a.b.c` in eslisp code, use the
 [*eslisp-propertify*][10] user-macro.
 
-For *computed* property access, omit the leading colon.
```

Comment:
This is probably better in another PR. It's an oversight independent of this PR, and is better judged separately.